ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml

```diff
@@ -596,7 +596,7 @@ objects:
         elif [[ "${CLUSTER_TYPE}" == "gcp" ]]; then
             # HACK: try to "poke" the token endpoint before the test starts
             for i in $(seq 1 30); do
-              code="$( curl -s -o /dev/null -w "%{http_code}" https://oauth2.googleapis.com/token -X POST -d '' )"
+              code="$( curl -s -o /dev/null -w "%{http_code}" https://oauth2.googleapis.com/token -X POST -d '' || echo "Failed to POST https://oauth2.googleapis.com/token with $?" 1>&2)"; then
```

Comment:
You still have a then at the end of the line

Reply:
> You still have a then at the end of the line

:man_facepalming:  fixed with d612a99f1 -> 2336b88da.